doc/conf.py

```diff
@@ -291,7 +291,7 @@ def setup(app):
     # Do not pop up any mayavi windows while running the
     # examples. These are very annoying since they steal the focus.
     mlab.options.offscreen = True
-except ImportError:
```

Comment:
I had to have it in order to test on my system, so yes please put it in `master`, I can open another PR, or just modify it here if you don't mind having the orthogonal change here